extensions/multiline.md

```diff
@@ -54,6 +54,8 @@ The only defined capability key so far is:
 * `max-bytes` - This defines the maximum allowed total byte length of multiline batched content *REQUIRED*
```

Comment:
Suggestion: "This defines the maximum allowed total byte length of a multiline batch's message value, as defined below"